utils/products.js

```diff
@@ -1,152 +1,372 @@
+    productDescription:
+      "The project provides core services for querying, adding, and changing business partner data in the Catena-X data space. Currently, BPDM consists of the Pool and Gate API.",
     githubRepo: [
-      "https://github.com/eclipse-tractusx/bpdm"
+      "https://github.com/eclipse-tractusx/bpdm",
+      "https://github.com/eclipse-tractusx/bpdm-certificate-management",
     ],
     committers: [
       "https://github.com/nicoprow",
-      "https://github.com/pazepaze",
-      "https://github.com/Maximilianong"
+      "https://github.com/SujitMBRDI",
+      "https://github.com/Maximilianong",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Business Partner Data Management Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "bpn-did-resolution-service",
+    productDescription: "Tractus-X Resolver Service for BPN <> DID resolution",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/bpn-did-resolution-service",
+    ],
+    committers: [
+      "https://github.com/paullatzelsperger",
+      "https://github.com/wolf4ood",
+      "https://github.com/jimmarino",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request bpn-did-resolution-service Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "demand-capacity-mgmt",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/demand-capacity-mgmt"],
+    committers: ["https://github.com/nitin-vavdiya"],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request demand-capacity-mgmt Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Digital Product Pass",
+    productDescription: 
+    "The Digital Product Pass (DPP) Application provides an easy way to request and visualize product passports from an economic operator using the Catena-X Network. By scanning a QR code or introducing a manufacturerPartId and partInstanceId (productId) passports can be displayed for different products like Batteries (Battery Pass), Transmissions (Transmision Pass) and any other product by simply using the Generic Digital Product Passport Model. Additionally components like the dpp-backend have the power to retrieve any aspect submodel content, which is registered in a Digital Twin using the Catena-X Network, and the dpp-verification add-on enables certification + verification of Catena-X Instance/Type Aspect Models",
+    githubRepo: ["https://github.com/eclipse-tractusx/digital-product-pass"],
+    committers: [
+      "https://github.com/matbmoser",
+      "https://github.com/saudkhan116",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Business Partner Data Management Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request digital-product-pass Team",
+    hasBoard: true,
+    showVersion: true,
+  },
+  {
```

Comment:
This product forwards to another [github project & repo](eclipse-dataspace-dcp/decentralized-claims-protocol), likely as this has been moved to be handled by the EF working groups more independent of Eclipse Tractus-X.

Recommend removing

Reply:
Yes, should be removed, https://github.com/eclipse-tractusx/identity-trust moved to https://github.com/eclipse-dataspace-dcp/decentralized-claims-protocol